utils_nlp/models/transformers/common.py

```diff
@@ -58,35 +57,31 @@ def __init__(
         cache_dir=".",
         load_model_from_dir=None,
     ):
-        self.model_name = model_name
+
+        if model_name not in self.list_supported_models():
+            raise ValueError(
+                "Model name {0} is not supported by {1}. "
+                "Call '{2}.list_supported_models()' to get all supported model "
+                "names.".format(value, self.__class__.__name__, self.__class__.__name__)
```

Comment:
This was a mistake I made. We only need to keep self.\_\_class\_\_.\_\_name\_\_ and change {2} to {1}.